src/app/researcher/study/[encodedStudyId]/review/runs-table.tsx

```diff
@@ -33,17 +31,16 @@ export type Study = {
 }
 
 type RunsTableProps = {
-    // studyIdentifier: string
     encodedStudyId: string
     isActive: boolean
     study: Study
 }
 
-const RunsTable: React.FC<RunsTableProps> = ({ encodedStudyId, isActive, study }) => {
+const RunsTable: React.FC<RunsTableProps> = ({ isActive, study }) => {
     const queryClient = useQueryClient()
-    const [viewingRunId, setViewingRunId] = useState<string | null>(null)
+    const [__, setViewingRunId] = useState<string | null>(null)
```

Comment:
`__` variable. Maybe explain it more?

Reply:
no, `_` is just a "unused var" that is exluded from aslant rules. I need to figure out what's going on with that setState since it doesn't appear used at all